src/components/WarningModal.tsx

```
@@ -0,0 +1,61 @@
+
+interface WarningModalProps {
+	totalBalance: bigint;
+	threshold?: number;
+	isOpen?: boolean;
+	onClose?: () => void;
+	network: NetworkConfig;
+}
+
+export function WarningModal({ 
+	totalBalance,
+	isOpen = false,
```

ESLint

[error] 'isOpen' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)